src/textual/widgets/_progress_bar.py

```diff
@@ -2,22 +2,43 @@
 
 from __future__ import annotations
 
-from math import ceil
+from math import ceil, cos, radians
 from time import monotonic
 from typing import Callable, Optional
+from typing_extensions import Literal
 
 from rich.style import Style
 
 from textual.geometry import clamp
 
 from ..app import ComposeResult, RenderResult
 from ..containers import Horizontal
+from ..css._error_tools import friendly_list
 from ..reactive import reactive
 from ..renderables.bar import Bar as BarRenderable
 from ..timer import Timer
 from ..widget import Widget
 from ..widgets import Label
 
+from ..color import Color
+
+
+BarColorScheme = Literal["default", "rainbow"]
+"""The names of the valid bar color_shemes.
+
+These are the color_shemes that can be used with a [`Bar`][textual.widgets.Bar].
```

Comment:
Either use the English words or use backticks to style attributes/parameters/code entities.

For example, in English:

```suggestion
"""The names of the valid bar color schemes.

These are the color schemes that can be used with a [`Bar`][textual.widgets.Bar].
```

Reply:
Fixed.